source/dub/dub.d

```diff
@@ -1145,21 +1145,24 @@ class Dub {
 		initPackage(path, depVers, type, format, recipe_callback);
 
 		if (!["vibe.d", "deimos", "minimal"].canFind(type)) {
-			runCustomInitialization(path, type, app_args);
+			runCustomInitialization(path, type, typeVersion, app_args);
 		}
 
 		//Act smug to the user.
 		logInfo("Success", Color.green, "created empty project in %s", path.toNativeString().color(Mode.bold));
 	}
 
-	private void runCustomInitialization(NativePath path, string type, string[] runArgs)
+	private void runCustomInitialization(NativePath path, string type, string typeVersion, string[] runArgs)
 	{
 		string packageName = type;
-		auto template_pack = m_packageManager.getBestPackage(packageName);
-		if (!template_pack) template_pack = m_packageManager.getBestPackage(packageName, "~master");
+		Package template_pack;
+		VersionRange versionRange = VersionRange.Any;
+		if (typeVersion)
```

Comment:
```suggestion
		if (typeVersion.length)
```

because typeVersion comes from the public facing API where an empty string might be passed in (which evaluates to true)